lms/djangoapps/student_account/views.py

```diff
@@ -133,9 +133,7 @@ def login_and_registration_form(request, initial_mode="login"):
             'third_party_auth_hint': third_party_auth_hint or '',
             'platform_name': configuration_helpers.get_value('PLATFORM_NAME', settings.PLATFORM_NAME),
             'support_link': configuration_helpers.get_value('SUPPORT_SITE_LINK', settings.SUPPORT_SITE_LINK),
-            'password_reset_support_link': configuration_helpers.get_value(
-                'PASSWORD_RESET_SUPPORT_LINK', settings.SUPPORT_SITE_LINK  # Intentional default.
-            ),
```

Comment:
I believe we should be using `configuration_helpers.get_value(...)`, so this looks wrong to me. Do you know why it wasn't working? I think the issue is that you put 

```python
            'password_reset_support_link': configuration_helpers.get_value(
                'PASSWORD_RESET_SUPPORT_LINK', settings.SUPPORT_SITE_LINK  # Intentional default.
            ),
```

but you should put

```python
            'password_reset_support_link': configuration_helpers.get_value(
                'PASSWORD_RESET_SUPPORT_LINK', settings.PASSWORD_RESET_SUPPORT_LINK
            ),
```

The default of `PASSWORD_RESET_SUPPORT_LINK` being `SUPPORT_SITE_LINK` is already defined in `aws.py` so it's not DRY to repeat it here anyways (though some environments like tests do not use `aws.py`). But if you do need to specify the default here as well, then I believe you need to do:

```python
            'password_reset_support_link': (
                configuration_helpers.get_value(
                    'PASSWORD_RESET_SUPPORT_LINK', settings.PASSWORD_RESET_SUPPORT_LINK
                ) or settings.SUPPORT_SITE_LINK
            )
```